js/okx.js

```diff
@@ -3963,7 +3963,7 @@ module.exports = class okx extends Exchange {
             const transfer = data[i];
             resultArray.push (this.parseTransfer (transfer, undefined));
         }
-        return this.filterBySinceLimit (resultArray, since, limit);
+        return this.filterBySinceLimit (resultArray, undefined, undefined);
```

Comment:
We are not expecting an array from a method that returns a single transfer by id.